wormhole-connect/src/views/v2/Bridge/Routes/SingleRoute.tsx

```diff
@@ -264,22 +264,28 @@ const SingleRoute = (props: Props) => {
   }, [quote]);
 
   const routeCardHeader = useMemo(() => {
-    return typeof receiveAmount === 'undefined' ? (
-      <CircularProgress size={18} />
-    ) : (
+    if (props.error) {
+      return <Typography color="error">Route is unavailable</Typography>;
+    }
+
+    if (typeof receiveAmount === 'undefined') {
+      return <CircularProgress size={18} />;
```

Comment:
If quote fails without an error, this may get stuck here. Should we check `isFetchingQuote` instead?